File: microservicio-cuenta-movimientos/src/main/java/com/develop/cuentamovimientos/dto/ClienteDTO.java
package com.develop.cuentamovimientos.dto;

import com.develop.cuentamovimientos.enums.GenderPerson;
import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ClienteDTO implements Serializable {
    private Long id;
    private LocalDateTime creationDate;
    private String fullName;
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private GenderPerson genderPerson;
    private int age;
    private String identification;
    private String address;
    private String phone;
    private String password;
    private boolean state;
}

File: client-person/Dto/ClienteDto.cs
using System.Text.Json.Serialization;
using client_person.Enum;

namespace client_person.Dto
{
    [JsonUnmappedMemberHandling(JsonUnmappedMemberHandling.Disallow)]
    public class ClienteDto
    {
        public long id { get; set; }
        public DateTime creationDate { get; set; }
        public string? fullName { get; set; }
        public GenderPerson genderPerson { get; set; }
        public int age { get; set; }
        public string? identification { get; set; }
        public string? address { get; set; }
        public string? phone { get; set; }
        public string? password { get; set; }
        public bool state { get; set; }
        
    }
}